server/api/[...].js

```diff
@@ -141,6 +137,4 @@ router.delete(
   }
 );
 
-app.use("/api", router);
-
-app.listen(BACKEND_PORT);
+export default useBase("/api", fromNodeMiddleware(app));
```

Comment:
> なお、router を使っているのは自分が扱うパスが /api などの配下だけであると分かり易いコードを書くためです。今のパッチでは router 使わず app 使って最後に useBase する形にされてますがこちらみたいな書き方の方が見通しやすい＆ express を別起動するときとコードの変化も少なくて良さそうに思います。

_Originally posted by @dynamis in https://github.com/webdino/lyceum-pokemon/issues/119#issuecomment-1450833442_